torchvision/datasets/triplet.py

```diff
@@ -0,0 +1,86 @@
+import torch
+import torch.utils.data as data
+
+import math
+
+
+def get_class_samples(num_classes, samples):
```

Comment:
is `num_classes` needed? Can't we instead make `class_samples` be a `defaultdict(list)`?

Reply:
Yes we can. I didn't know about this function. Thanks for introducing it to me.

I have made this change. The method looks a lot cleaner now.